src/mod/l4_thruster.c

```diff
@@ -190,295 +151,23 @@ void l4_thruster_main (void) {
 
 
 void l4_debugPrintBuffer(uint8_t *buffer,int bufferlen){
-
 	//LPC_UART2 is debug UART
-
 	for (int i=0;i<bufferlen;i++){
 		Chip_UART_SendByte(LPC_UART2, buffer[i]);
-
 	}
 
 }
 
 
 
-/////  BELOW WOULD BE FUNCTIONS TO CONTROLL THRUSTER
-
-
-
-
-void SetReservoirTemperature(int argc, char *argv[]){
```

Comment:
All manual request functions are deleted. They all can be implemented with general read/set request functions

Except for request to all registers at once